blazor/installation/install-nuget-packages.md

```diff
@@ -84,7 +84,7 @@ The **Package Manager Console** saves NuGet packages installation time since you
 
     **For example:** Install-Package Syncfusion.Blazor.Grid
 
-    N> You can find the list of Syncfusion Blazor NuGet packages which are published in nuget.org from [here](https://www.nuget.org/packages?q=Tags%3A%22Blazor%22+syncfusion)
+    N> You can find the list of Syncfusion Blazor NuGet packages which are published in nuget.org from [here](https://www.nuget.org/packages?q=Tags:%22Blazor%22+syncfusion)
```

Comment:
why this link changed? this link is not redirected.

Reply:
changed